### PR TITLE
pyramid test case using pytest fixture

### PR DESCRIPTION
**Pull request by petrus-v**

### CODE REVIEW

File: {{cookiecutter.project_slug}}/{{cookiecutter.python_package}}/{{cookiecutter.blok_name}}/tests/test_pyramid.py [hunk 1]
```diff
@@ -1,23 +1,22 @@
-from anyblok_pyramid.tests.testcase import PyramidBlokTestCase
 
 
-class TestCanigooRestApi(PyramidBlokTestCase):
+class TestCanigooRestApi:
```
Comment (jssuzanne) [inline {{cookiecutter.project_slug}}/{{cookiecutter.python_package}}/{{cookiecutter.blok_name}}/tests/test_pyramid.py]: Maybe, you should add fixture rollback_registry on the class be sur that

1. registry exist
2. registry is rollbacked between each test

Comment (petrus-v) [reply {{cookiecutter.project_slug}}/{{cookiecutter.python_package}}/{{cookiecutter.blok_name}}/tests/test_pyramid.py]: oh yes you're right !

Comment (petrus-v) [reply {{cookiecutter.project_slug}}/{{cookiecutter.python_package}}/{{cookiecutter.blok_name}}/tests/test_pyramid.py]: @jssuzanne, I'm not that aware about pytest is the [class decorator](https://github.com/AnyBlok/cookiecutter-anyblok-project/pull/23/commits/2aa78531fc82e6158bbc58ef89c2b1991166be9f) enough ?

Comment (petrus-v) [reply {{cookiecutter.project_slug}}/{{cookiecutter.python_package}}/{{cookiecutter.blok_name}}/tests/test_pyramid.py]: it looks like :) https://docs.pytest.org/en/2.8.7/fixture.html#using-fixtures-from-classes-modules-or-projects

**Review by jssuzanne — APPROVED** [review]

**Review by jssuzanne — APPROVED** [review]

**Pull request merged**